scripts/entity/components/movement_components/base_movement_component.gd

```diff
@@ -92,7 +92,7 @@ func on_jump_input_cancelled() -> void:
 ## Calls the movement signals using the given velocity and previous velocity
 func _emit_movement_signals() -> void:
 	if(velocity.x == 0 && _previous_horizontal_velocity != 0):
-		on_horizontal_movement_cancelled.emit(velocity.x)
+		on_horizontal_movement_cancelled.emit()
```

Comment:
Why did this lose its param?

Reply:
We talked about this in vc. It only ever emits when velocity is 0, so it doesn't need a param.